### ISSUE TITLE
Is it the know jitter issue?

### ISSUE THREAD

**Issue by pthag** [janpaepke/ScrollMagic#310]
I have a really basic animation. Just a center logo that scales with the scroll position and gets pinned at the top.

I've tried several approaches with GSAP and ScrollMagic, but it still looks like there is some calculation issue. I first was thinking maybe it's just because I used an svg, but with an png its just the same. And then I tried and read a lot how to tackle this problem, but its still there. Maybe its just the same as the know issue with jitter, but I don't know for sure. If it is let me know.

In my opinion it looks like GSAP is doing an animation but then the css settings is repositioning it afterwards again. But I tried it with out the css center trick with margin minus and still the same weird behaviour.

Or is it when I'm heading to the end of the animation curve gets so flat that is behaving this way cause its going reaaalllyyy slow.

I'll add in a recreation of my problem in a few minutes.

**Comment by pthag** [comment]
I used a TimelineMax in a variabel. With this animation. 

.to(".the-logo-image", 1, {top: "75px", width: "500px", marginLeft: "-250px"}, 0.12) 

Nothing special I would say?

**Comment by pthag** [comment]
Its a site still under development, but you can see the issue at new.wunderhund.nl

**Comment by janpaepke** [comment]
Well you haven't actually said what your issue is...

But from checking out the demo (please make sure it stays available!) my guess is you mean that the animation of the Logo doesn't appear to be smooth.

#### Analysis
This is actually due to the way you set up the animation. You size it and move it at the same time and you use `width` and `margin-left` to do that. Yet during the animation only full px values are used.
I don't think that's GSAP default behavior and might be due to the fact that you supplied strings instead of integers (`top: "75px"` instead of `top: 75`).

Anyways: Open your element inspector and check how the values change.
Your intention is that the margin is always half of the width, to center the image.
So your plan is this:
500 -> 250
501 -> 250.5
502 -> 251
503 -> 251.5
When in reality this is not happening, for above mentioned reasons.
This is what makes the animation seem "unsmooth", especially, as it gets slower.

What I can say for sure though, is that it has nothing to do with the known jitter issue.

Also: What is the `0.12` at the end supposed to do? [`TweenMax.to()`](http://greensock.com/docs/#/HTML5/GSAP/TweenMax/to/) only accepts 3 parameters...

#### Resolution
Use transforms instead. They are [much better performance wise](https://www.youtube.com/watch?v=R6TXuXV1bbY) and are float vals by default.
Also if you use `scale` the transform origin is usually the center, so you don't have to take care of centering it manually.

Bottom line – try this:
```js
TweenMax.to(".the-logo-image", 1, {y: 75, scale: 0.5})
```

**Comment by pthag** [comment]
Thanks a lot for the very good explanation. 

The 0.12 is just an offset of the startpoint when the animation has to start. So it starts after 0.12 of the duration which is 1. That is working well. I found out this feature actually makes it possible to animate beyond the ending of the ScrollerMagic scene duration.

I first had the scale approach. Only the thing is I have 3 different sizes defined in @media screen rules as a starting point. So this way they always end up in the same 500px width. Which is required.

**Comment by janpaepke** [comment]
This means you're using TimelineMax instead of TweenMax.
Instead of adding the 0.12 and using TimelineMax you can also add `{delay: 0.12}` to your TweenMax tween.

Can I consider this issue as resolved?

**Comment by pthag** [comment]
Yes, tried a bit l'll just make a script to specify the scale size depending on the screen size.

**Comment by janpaepke** [comment]
all right. then.
remember to keep the demo available.

**Comment by pthag** [comment]
I changed the demo in a jsfiddle.

**Comment by janpaepke** [comment]
thx but it doesn't work.

**Comment by pthag** [comment]
Sorry, I had the impression you mean keeping the demo online what was going wrong, not a working solved example.

**Comment by janpaepke** [comment]
As per the [support guidelines](https://github.com/janpaepke/ScrollMagic/blob/master/CONTRIBUTING.md#support-guidelines) it's quite important for other users to be able to see a working demo. Otherwise they won't be able to learn anything from this.

But you should be able to make the demo functional, no?

**Comment by pthag** [comment]
I'll post fully working one when I fixed it all in a few minutes. And thanks for the great support!

**Comment by janpaepke** [comment]
you're welcome :)
